packages/opentelemetry-node/test/host-metrics.test.js

```diff
@@ -43,68 +44,38 @@ const testFixtures = [
         // verbose: true,
         checkTelemetry: (t, collector) => {
             const metrics = collector.metrics;
-            const networkMetrics = metrics.filter((metric) =>
+            const systemMetrics = metrics.filter((metric) =>
                 metric.name.startsWith('system.network')
```

Comment:
Perhaps this:

```suggestion
                metric.name.startsWith('system.')
```

? But it is fine as is.